### PR TITLE
bump: Update base image CY-5141

### PR DESCRIPTION
**Pull request by jonathancarvalheiro**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 target
 node_modules
 .bsp
+.metals
```
Comment (lolgab) [inline .gitignore:4]: Newline here

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-ARG alpine_version=3.13
+ARG alpine_version=3.13.6
```
Comment (lolgab) [inline Dockerfile:1]: We are bumping the other images to 3.14.2

**Review by lolgab — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (jonathancarvalheiro) [reply .gitignore]: Done

File: Dockerfile (hunk 2 above)
Comment (jonathancarvalheiro) [reply Dockerfile]: Done

**Review by lolgab — APPROVED** [review]

**Pull request merged**